cpu_linux.go

```diff
@@ -23,37 +23,33 @@
 package madmin
 
 import (
-	"github.com/prometheus/procfs/sysfs"
+	"os"
+	"path/filepath"
+	"strconv"
 )
 
-func getCPUFreqStats() ([]CPUFreqStats, error) {
-	fs, err := sysfs.NewFS("/sys")
-	if err != nil {
-		return nil, err
-	}
+func getCPUFreqStats() (stats []CPUFreqStats, err error) {
+	for i := 0; ; i++ {
+		governorPath := filepath.Join(
+			"/sys/devices/system/cpu",
+			"cpu"+strconv.Itoa(i),
```

Comment:
Might be better to store this in a variable `cpuName := "cpu"+strconv.Itoa(i)` and use in both the places.